src/GovUk.Education.ExploreEducationStatistics.Content.Api.Tests/Controllers/ReleaseFileControllerTests.cs

```diff
@@ -136,13 +122,28 @@ public async Task StreamFilesToZip_NoFileIds()
                     (_, stream, _, _) => { stream.WriteText("Test zip"); }
                 );
 
+            var client = SetupApp(releaseFileService: releaseFileService.Object)
+                .AddContentDbTestData(context => context.Releases.Add(_release))
+                .CreateClient();
+
             var response = await client
                 .GetAsync($"/api/releases/{_release.Id}/files");
 
-            MockUtils.VerifyAllMocks(_releaseFileService);
+            MockUtils.VerifyAllMocks(releaseFileService);
+
+            response.AssertOk("Test zip");
+        }
 
-            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
-            Assert.Equal("Test zip", await response.Content.ReadAsStringAsync());
+        private WebApplicationFactory<TestStartup> SetupApp(IReleaseFileService? releaseFileService = null)
+        {
+            return _testApp
+                .ResetDbContexts()
+                .ConfigureServices(
+                    services =>
+                        {
+                            services.AddTransient(_ => releaseFileService ?? Mock.Of<IReleaseFileService>());
+                        }
+                    );
```

Comment:
Although the test cases currently do nothing else currently, we may need to add further test cases where we actually do some further setup so I'd prefer to keep this the way it is.

This would be consistent with the general pattern of having a `SetupApp` method (which is analogous to the `Build/SetupService` pattern we use in service tests).